canonical-websites/snapcraft.io
# Filter unreleased revisions

Part of #1473

First part of work on selecting available revisions.
- adds simple dropdown with list of filters (Recent, Unreleased, All)
- choosing one of the then filters the list of available revisions that can be selected
- as 'All' revisions filter are now part of the dropdown in releases table, 'Show all revisions...' is removed from below the table

Demo: https://snapcraft-io-canonical-websites-pr-1478.run.demo.haus/

### QA
- ./run or https://snapcraft-io-canonical-websites-pr-1478.run.demo.haus/
- go to Releases page of any snap
- 'Unassigned' row should be separated from the rest of the page with dropdown available
- you should be able to select filter to revisions (Recent, Unreleased and All)
- list of releases available to select should change when filter is changed

More functionality and visual updates will be added as follow up.

<img width="1028" alt="screen shot 2018-12-20 at 11 24 25" src="https://user-images.githubusercontent.com/83575/50279400-dab61300-0449-11e9-9232-2d20fae64797.png">
